hisensetv/__main__.py

```diff
@@ -51,6 +51,22 @@ def main():
             "netflix",
             "youtube",
             "amazon",
+            "disney+",
+            "hulu",
+            "peacock_tv",
+            "tubi",
+            "pluto_tv",
+            "vudu",
+            "hbo_max",
+            "youtube_music",
+            "iheartradio",
+            "pandora",
+            "tidal",
+            "spotify",
+            "tunein_radio",
+            "plex",
+            "mrmc", 
```

Comment:
```suggestion
            "mrmc",
```

There's a trailing space there.

Reply:
Thanks.
have no idea how to use GIT, so used notepad++ to find all the spaces and tabs and linefeeds I put in by accident.
I then copy/pasted what worked at home over.